docs/reference/concepts/03-evaluation-context.mdx

```diff
@@ -102,6 +102,24 @@ evalCtx := openfeature.NewEvaluationContext(
 boolValue, err := client.BooleanValue("boolFlag", false, evalCtx)
 ```
 
+</TabItem>
+<TabItem value="php" label="PHP">
+
+```php
+// add a value to the global context
+$api = OpenFeatureAPI.getInstance();
+$api.setEvaluationContext(new EvaluationContext("targetingKey", ["myGlobalKey" => "myGlobalValue"]));
```

Comment:
Is the `targetingKey` always required when constructing the evaluation context?

Reply:
It can also be null. I have seen some of the SDKs are using the Builder pattern but because there are so few parameters it's actually more overhead to create a builder than simply passing `null`.

```php
(new EvaluationContextBuilder()).setAttributes(["key" => "value"]).build()

// vs

new EvaluationContext(null, ["key" => "value"]);
```